braindecode/models/fbcnet.py

```diff
@@ -0,0 +1,223 @@
+
+    Parameters
+    ----------
+    n_bands : int or None or List[Tuple[int, int]]], default=9
+        Number of frequency bands. Could
+    n_filters_spat : int, default=32
+        Number of spatial filters for the first convolution.
+    n_dim: int, default=3
+        Number of dimensions for the temporal reductor
+    temporal_layer : str, default='LogVarLayer'
+        Type of temporal aggregator layer. Options: 'VarLayer', 'StdLayer',
+        'LogVarLayer', 'MeanLayer', 'MaxLayer'.
+    stride_factor : int, default=4
+        Stride factor for reshaping.
+    activation : nn.Module, default=nn.SiLU
+        Activation function class to apply in Spatial Convolution Block.
+    filter_parameters: dict, default {}
+        Parameters for the FilterBankLayer
+
+    References
+    ----------
+    .. [fbcnet2021] Mane, R., Chew, E., Chua, K., Ang, K. K., Robinson, N.,
+        Vinod, A. P., ... & Guan, C. (2021). FBCNet: A multi-view convolutional
+        neural network for brain-computer interface. preprint arXiv:2104.01233.
+    .. [fbcnetcode2021] Link to source-code:
+        https://github.com/ravikiran-mane/FBCNet
+    """
+
+    def __init__(
+        self,
+        # Braindecode parameters
+        n_chans=None,
+        n_outputs=None,
+        chs_info=None,
+        n_times=None,
+        input_window_seconds=None,
+        sfreq=None,
+        # models parameters
+        n_bands=9,
+        n_filters_spat: int = 32,
+        temporal_layer: str = "LogVarLayer",
+        n_dim: int = 3,
+        stride_factor: int = 4,
+        activation: nn.Module = nn.SiLU,
+        filter_parameters: dict = {},
+    ):
+        super().__init__(
+            n_chans=n_chans,
+            n_outputs=n_outputs,
+            chs_info=chs_info,
+            n_times=n_times,
+            input_window_seconds=input_window_seconds,
+            sfreq=sfreq,
+        )
+        del n_outputs, n_chans, chs_info, n_times, input_window_seconds, sfreq
+
+        # Parameters
+        self.n_bands = n_bands
+        self.n_filters_spat = n_filters_spat
+        self.n_dim = n_dim
+        self.stride_factor = stride_factor
+        self.activation = activation
+        self.filter_parameters = filter_parameters
+
+        # Checkers
+        if temporal_layer not in _valid_layers:
+            raise NotImplementedError(
+                f"Temporal layer '{temporal_layer}' is not implemented."
+            )
+
+        if self.n_times % self.stride_factor != 0:
+            warn(
+                f"Time dimension ({self.n_times}) is not divisible by"
+                f" stride_factor ({self.stride_factor}). Input will be padded.",
+                UserWarning,
+            )
+
+        # Layers
+        # Following paper nomenclature
+        self.spectral_filtering = FilterBankLayer(
+            n_chans=self.n_chans,
+            sfreq=self.sfreq,
+            band_filters=self.n_bands,
+            verbose=False,
+            **filter_parameters,
```

Comment:
```suggestion
            **(filter_parameters or {}),
```
handle None case here